models/pet.js

```diff
@@ -25,6 +25,10 @@ module.exports = (sequelize, DataTypes) => {
       type: DataTypes.STRING,
       allowNull: false
     },
+    dob: {
```

Comment:
Not the right Syntax.  Next time try this.  I have already fixed it in my branch

    dob: {
      type: DataTypes.STRING,
      allowNull: false,
      validate: {
        isDate: true
      }
    }